examples/normal-usage.js

```diff
@@ -29,8 +29,8 @@ console.log('Background color attack!'.black.bgWhite);
 console.log('Use random styles on everything!'.random);
 console.log('America, Heck Yeah!'.america);
 
-console.log('Blindingly '.brightCyan + 'bright? '.brightRed
-            + 'Why '.brightYellow + 'not?!'.brightGreen);
+console.log('Blindingly '.brightCyan + 'bright? '.brightRed +
+            'Why '.brightYellow + 'not?!'.brightGreen);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/labspl/colors.js/pullRequest?prid=8695913)
```suggestion
            "Why ".brightYellow + 'not?!'.brightGreen);
```